lib/generateMarkdown.js

```diff
@@ -87,16 +87,32 @@ function createPropertiesSummary(schema, suppressWarnings) {
     md += '|   |Type|Description|Required|\n';
     md += '|---|----|-----------|--------|\n';
 
-    var properties = schema.properties;
-    for (var name in properties) {
-        if (properties.hasOwnProperty(name)) {
-            var property = properties[name];
-            var summary = getPropertySummary(property, suppressWarnings);
+    var propertiesKeys = ['properties', 'patternProperties'];
```

Comment:
Ya, I was actually hoping that would be merged.  I didn't want to reinvent the wheel, but I needed a solution.  One way or another, wetzel needs support for `patternProperties` - I don't care if it's mine or PR #10 , but getting one of these solutions implemented is essential.